example_agents/acme_r2d2/network.py

```diff
@@ -6,11 +6,13 @@ class R2D2Network:
     def __init__(self):
         self.rnn = BasicRNN(self.environment)
 
-    def restore(self):
-        self.AcmeRun.restore_tensorflow("rnn", self.rnn)
+    def save(self, run=None):
```

Comment:
This appears to not-quite-work for acme_r2d2 because you need to pass the run when you call `save()` (in agent.py and trainer.py) for the model to actually be logged in the run so it can be restored in a later run.  Seems to work once I pass the run!

Reply:
Fixed.